src/pymovements/plotting/heatmap.py

```diff
@@ -49,7 +51,10 @@ def heatmap(
         path_to_image_stimulus: str | Path | None = None,
         stimulus_origin: str = 'upper',
         alpha: float = 1.,
-) -> plt.Figure:
+        *,
+        ax: plt.Axes | None = None,
+        closefig: bool | None = None,
+) -> tuple[plt.Figure, plt.Axes]:
```

Comment:
It would be great if we could manage to avoid this breaking change. How could we achieve that?

Reply:
We would probably need an additional argument (like `return_only_figure: bool = True`), that would raise a deprecation warning if `True`, with the instruction to set it to `False`. Then after a few minor versions we would remove this altogether.

So the perfect process would be something like:
1. add a `return_only_figure` argument with a deprecated default value (`True`).
2. change the default value to `False` and deprecate the argument.
3. remove the argument and remove the union from the return typehint.

I think this would be very inconvenient for users, as they would need to fiddle with this argument at least two times.

As this is just a plotting function, I would also be ok with having this breaking change.

Reply:
I think breaking here would be fine, especially if we at the same time update the tutorials, i.e. before next release -- is this possible?